### ISSUE TITLE
evaluate model performance in 'eval' mode

### ISSUE THREAD

**Issue by Tae39** [google-research/simclr#82]
Hi，Sorry to bother you. When I set mode as 'eval', I found that label_top_1_accuracy and label_top_5_accuracy were always 1 of different models, so I can't evaluate model performance without using 'train_then_eval', any idea where I went wrong? Moreover, I'm not quite sure the difference between 'contrast_loss', 'loss' and 'regularization_loss'. Look forward to your kind reply.

**Comment by chentingpc** [comment]
Did you set `--train_mode=finetune`?

`contrast_loss` is from nt-xent loss. `regularization_loss` is the l2 loss of weights. `loss` is total loss summing over all of them. Note that the `contrast_loss` is dependent on batch size, so training and eval time it may be different (as bsz may be different).

**Comment by Tae39** [comment]
Thank you so much and I've got that! 
 And there is another place in paper 2002 I want to confirm with you. Is the main difference between 'linear evaluation' and 'fine_tuned' is whether to use a 'linear projection head'? Sincere thanks again.

**Comment by Tae39** [comment]
Thank you so much and I've got that!
And there is another place in paper of SimCLR I want to confirm with you. Is the main difference between 'linear evaluation' and 'fine_tuned' is whether to use a 'linear projection head'? Sincere thanks again.

**Comment by Tae39** [comment]
Thank you so much and I've got that!
